### PR TITLE
Taj Hairstyles - Mostly Variations & Tweaks With Some Additions

### PR DESCRIPTION
**Pull request by WickedCybs**
Received some suggestions on adding more variations of what we have, so I decided to go for it because variety is the spice of life. Additionally, a lot of the alt hairstyles currently had a totally different style and these have been renamed to shoulderparted/sideparted to give people a better idea of what is similar to what.

This will mean anyone using those hairstyles will need to reselect, but it's a worthy sacrifice to make the names understandable. 

Adds two new hairstyles in total, fringe up and cascading. Around six or so new variations with stuff like a longer and shorter variant of tresses alt (now taj shoulderparted), sideparted with the option to have it on either shoulder and a little more.

Also adds a new Taj preview that comes with all 4 directions. Added it for my sake and so others don't have to piece together all the limbs when it comes to future PRs.

### CODE REVIEW

**Comment by WickedCybs** [comment]
I'm away from my main computer right now, the PR is essentially almost done but I don't think I'm comfortable with having it ready for approval yet.

**Comment by WickedCybs** [comment]
Actually I think there's not really any big changes I'd do at this point. So it might as well be good to review.

**Comment by WickedCybs** [comment]
!review

File: html/changelogs/wickedcybs_tajhair.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+author: WickedCybs
+
+delete-after: True
+
+changes:
+  - rscadd: "Added some new Taj hairstyles and variations. Mainly variations."
+  - rscadd: "A second preview that can be viewed in all different directions has been added to the tajara dmi. This is only really of interest to spriters."
+  - tweak: "A lot of the current taj alt hairs had their names changed to make it more clear what is similar in style"
```
Comment (Casper3667) [inline html/changelogs/wickedcybs_tajhair.yml:8]: Since people will need to reselect the styles, this should be mentioned in the changelog

**Review by Casper3667 — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Comment by KasparoVy** [comment]
I really dig the new styles, cheers!

**Pull request merged**